packages/mask/src/plugins/Pets/SNSAdaptor/ModelView.tsx

```diff
@@ -0,0 +1,40 @@
+import type { FC, HTMLProps } from 'react'
+import '@webcomponents/custom-elements'
+import '@google/model-viewer/dist/model-viewer'
+import { Punk3D } from '../constants'
+
+interface ModelViewElementProps extends HTMLProps<HTMLDivElement> {
+    'shadow-intensity': string
+    'camera-controls': boolean
+    'auto-rotate': boolean
+    ar: boolean
+}
+
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            'model-viewer': any
```

Comment:
You forget to use it `ModelViewElementProps`

```suggestion
            'model-viewer': ModelViewElementProps
```

Reply:
in the newest pull, model-viewer have removed, maybe github not refresh?

Reply:
I see.